recipes/wayland-protocols/all/conanfile.py

```diff
@@ -0,0 +1,52 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import os
+
+
+class WaylandProtocolsConan(ConanFile):
+    name = "wayland-protocols"
+    description = "The Wayland-Protocols package contains additional Wayland protocols that add functionality outside of protocols already in the Wayland core."
+    topics = ("conan", "wayland-protocols", "wayland")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://wayland.freedesktop.org/"
+    license = "MIT"
+    generators = "pkg_config"
+
+    settings = []
+    options = {}
+    default_options = {}
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    _autotools = None
+
+    build_requires = (
+        "wayland/1.19.0"
+    )
```

Comment:
This is not needed to build (at least using Meson) although the build-system declares the dependency.